README.md

```diff
@@ -73,7 +73,7 @@ Here a configuration example:
 
 ```yaml
 m6_web_amqp:
-    sandbox: false
+    sandbox: false #optional - default false
```

Comment:
We could have an option to specify a file containing arbitrary data, for debug/test purpose, like this:
```yaml
sandbox:
    enabled: true
    file: mock.json
```
```json
[
    {
        // data
    },
    {
        // data
    }
]
```
But we do not want to version these kind of test data. IMHO, the use case is very specific, and your modifications are a first step. So this is enough for me, since we have the `enqueue` method, we can imagine more complexe scenarios now.